test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -2344,6 +2349,259 @@ public async Task ActorOrchestration_StringStoreWithCreateDelete(bool extendedSe
             }
         }
 
+        /// <summary>
+        /// End-to-end test which validates simple actor scenario.
+        /// </summary>
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory + "_UnpublishedDependencies")]

```

Comment:
This new category will not be executed in the CI. Is that your intention?

Reply:
Not sure. @ConnorMcMahon created this category in #664 and put the durable actor tests in it, so I thought I would use it for the new tests as well.

Reply:
I think it's basically a way to disable the test in the CI. The reason for this one in particular is because nuget dependencies were not yet available, either on nuget.org or myget.org. If they are available and you want these tests running, then we should remove the "_UnpublishedDependencies" suffix.